src/lambda_functions/receive_emails.py

```diff
@@ -0,0 +1,10 @@
+"""Receive Emails Lambda Function."""
+
+
+def lambda_handler(event, context):
+    """Lambda Handler."""
+    print("test")
```

Comment:
Since you're going to be making those other changes, you should change this to `print(event)`, then we can debug issues quicker with the terraform changes you'll be making.